City-of-Helsinki/helsinki-paatokset
# PP-199: Functionality for official decisions

This PR refactors the decision list functionality to work with decision nodes instead of custom entities.

**To test**
* Checkout branch
* SSH Into the container and run: `drush cim -y;drush cr;`
* If you don't have any decisions or decisionmakers, Run `drush ap:fs;drush mim ahjo_decisionmakers:all;drush mim ahjo_decisions:latest;drush ap:ud --localdata;drush cr`
* These decisionmakers should have decisions:
  * https://helsinki-paatokset.docker.so/fi/paattajat/asuntotuotantopaallikko-asuntotuotanto
  * https://helsinki-paatokset.docker.so/fi/paattajat/arviointitoiminnan-johtaja-selvitys-arviointi-ja-sijoitus 
* The latest two decisions should be visible on the decisionmaker page along with the content menu
* The "all decisions" and the "Decisions" link in the  subnavi should both work
* The decisions page should list all items for the decisionmaker, and the year tab navigation should work
